src/components/ContactsAdd.js

```diff
@@ -1,39 +1,156 @@
-import { useState } from "react"
-import { useNavigate } from "react-router-dom";
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
-function ContactsAdd(props) {
+function ContactsAdd({ contacts, setContacts }) {
+	const [addContactFormEmpty, setAddContactFormEmpty] = useState({
```

Comment:
I don't think this needed to be a state, as you don't need to reset the empty form ever. You could have made it a plain old javascript object.